lib/Dialect/EmitC/Conversion/MHLORegionOpsToEmitC.cpp

```diff
@@ -151,7 +151,7 @@ struct ConvertMhloRegionOpsToEmitCPass
     }
 
     OpBuilder builder(op);
-    auto ctx = op.getContext();
+    auto *ctx = op.getContext();
```

Comment:
We will most likely remove auto anyway :)

Reply:
Yes, it's just an intermediate change. I think it doesn't hurts.